esp32-controller/src/tasks/taskTelemetry/taskTelemetry.cpp

```diff
@@ -0,0 +1,16 @@
+#include "taskTelemetry.h"
+
+// Import External Variables //
+extern Ultrasonic ultrasonic;
+extern esp_websocket_client_handle_t ws_client;
+
+void taskTelemetry(void* pvParameters) {
+  for (;;) {
+    char* message = uint_to_str(ultrasonic.read());
+    esp_websocket_client_send_text(ws_client, message, strlen(message),
+                                   portMAX_DELAY);
```

Comment:
How do we identify the type of the message in the back-end?

Reply:
An identifier needs to be specified yet, but in the same way I mentioned in the EventHandler comment, I would like to discuss more about how we are doing the communication.